api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -305,4 +305,17 @@
 								 referencedTableName="concept"
 								 referencedColumnNames="concept_id" />
 	</changeSet>
+
+	<changeSet id="TRUNK-4505-20180804" author="fruether">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="person" columnName="cause_of_death_non_coded"/>
+			</not>
+		</preConditions>
+		<comment>Adding cause_of_death_non_coded to the table person</comment>
+		<addColumn tableName="person">
+			<column name="cause_of_death_non_coded" type="varchar(1024)" />
```

Comment:
Does this need to be this long? I'd imagine 255 should be fine

Reply:
There was no specification in the ticket description. I made it long to be sure al potential descriptions would fit.

I can change it to.254 or any other number if it is more applicable

Reply:
255 should be fine

Reply:
Then I will adapt it accordingly